applications/mne_scan/plugins/hpi/hpi.cpp

```diff
@@ -163,6 +164,9 @@ void Hpi::resetState()
 {
     m_bPluginControlWidgetsInit = false;
     m_pCircularBuffer->clear();
+
+    m_pFiffInfo.clear();
```

Comment:
When is the resetState() called. Please be aware that all plugins share one single fiff info. So clearing it here will also clear it for all other plugins.

Reply:
Wow.

Reply:
I am clearing the pointer instance, which just drops the reference by one.

Reply:
@gabrielbmotta ah ok. Now I see it too. That should be fine then :)
@juangpc Comments like "wow." are not very expressive. What would you like to say here?

Reply:
Yes. Sry for the noise. Deleted.